diesel/src/associations/mod.rs

```diff
@@ -149,6 +99,7 @@
 //! #   use users::dsl::*;
 //! #   let connection = establish_connection();
 //! #
+//! // Loading the posts for a single parent user.
 //! let user = users.find(1).first::<User>(&connection).expect("Error loading user");
```

Comment:
This example seems to be very similar to that one above, so do we really need both? (At least we could change this one to load all posts for multiple users?)